portlet-parent/wicketstuff-portlet/src/main/java/org/apache/wicket/portlet/PortletUrlRenderer.java

```diff
@@ -1,33 +1,57 @@
 package org.apache.wicket.portlet;
 
+import javax.portlet.PortletRequest;
+
 import org.apache.wicket.request.Request;
 import org.apache.wicket.request.UrlRenderer;
 import org.apache.wicket.util.lang.Args;
 import org.apache.wicket.util.string.PrependingStringBuffer;
 
 /**
- * {@link UrlRenderer} for Portlet Applications.
+ * <h1>{@link UrlRenderer} for Portlet Applications.</h1>
+ * <p>
+ * It corrects the base Url. All the generated URLs will be relative to this Url.
+ * </p>
+ * <p>
+ * The base Url depends on the lifecycle phase of the current portlet request.
+ * </p>
+ * <ul>
+ * <li>During the RENDER or the ACTION phase the base url must be set with the request url which is
+ * relative to the wicket filter path {@link Request#getUrl()}.  

```

Comment:
OK. This answers my question - there is no difference between RENDER and ACTION.